components/page-navigation/README.md

```diff
@@ -14,10 +14,10 @@ Do not use this component to replace or duplicate site navigation.
 
 ## Demo and sample markup
 
-<html-preview>
-
 ```html preview
-<cagov-page-navigation data-selector=".main" data-type="wordpress" data-label="On this page"></cagov-page-navigation>
+<sidebar>
```

Comment:
I'm late to the party but `<sidebar>` isn't an HTML tag as far as I'm aware. I think this should be `<aside>` instead. I know sidebar is in a number of our templates so rather than reopen I'll create an Issue for next sprint to address this.

Reply:
Yes, correct. Unfortunately, our CSS for the page-navigation component requires this use of `<sidebar>` at the moment. 

I've noted this in #672, and I see you created a new issue at #683 too. Wherever we address this, I agree it needs to be fixed.